### ISSUE TITLE
I get the following error 

### ISSUE THREAD

**Issue by arun-gautham** [qubvel/efficientnet#158]
module 'keras.api.backend' has no attribute 'sigmoid'

'2.15.0' is the TF version

'3.3.3' is the keras Version

**Comment by Entory520** [comment]
您好，我是恩泽。您发给我的信件已经收到。我会详细阅读并且尽快给您回复。如果超时未回复，请电话与我取得联系。
